cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/loadbalancer/AwsTargetGroup.java

```diff
@@ -49,7 +49,7 @@ public int getOrder() {
         return instanceIds;
     }
 
-    private static String getTargetGroupName(int port, AwsLoadBalancerScheme scheme) {
+    public static String getTargetGroupName(int port, AwsLoadBalancerScheme scheme) {
```

Comment:
Can this be package-private?

Reply:
No, it's referenced outside the package.